.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -1,67 +1,79 @@
   - type: textarea
-    id: desc
+    id: description
     attributes:
-      label: Description of issue
-      placeholder: The macguffin doesn't do anything when used.
+      label: Описание проблемы
+      description: В чем состоит суть проблемы? (Обязательное поле)
+      placeholder: Я сел на стул, от чего взорвался, а мой мозг оказался в душе на ЦК!
     validations:
       required: true
+
   - type: textarea
-    id: expected
+    id: what-expected
     attributes:
-      label: Difference between expected and actual behaviour
-      placeholder: The macguffin should kill the big bad when I use it.
+      label: Что должно было произойти?
+      description: Почему вы считаете это проблемой?
+      placeholder: Я ожидал, что я просто сяду на стул и буду сидеть.
+
   - type: textarea
-    id: repro-steps
+    id: what-happened
     attributes:
-      label: Steps to reproduce
-      description: Please provide a complete step-by-step method of reproducing the bug, including steps you may think are obvious. Also please make sure you have followed and verified these steps reproduce the bug yourself. Being verbose is better than being vague. If reproduction is unreliable, i.e., it only happens sometimes or only happened once, mention that in the reproduction steps.
-      placeholder: |
-        1.
-        2.
-        3.
-        ...
-    validations:
-      required: true
+      label: Что случилось вместо этого?
+      description: Как произошедшее отличается от ваших ожиданий?
+      placeholder: Я умер и оказался на ЦК.
+
   - type: textarea
-    id: paths
+    id: why-bad
     attributes:
-      label: Specific information for locating
-      description: e.g. an object name, specific message outputs, object paths or file names if known, specific maps and/or coordinates, etc
-      placeholder: |
-        Item name: The MacGuffin
-        Big bad name: Very Evil Mob
-        Item path: `/obj/item/macguffin`
+      label: Почему это плохо/Какие последствия?
+      description: Почему вы считаете эту проблему значительной?
+      placeholder: Игроки должны иметь возможность сидеть на стульях, не взрываясь.
+
   - type: textarea
-    id: version
+    id: how-to-reproduce
     attributes:
-      label: Client version, server revision, & game ID
-      description: This is found with the `Show server revision` verb in the OOC tab in game, or typing `show-server-revision` into the chat.
-      placeholder: |
-        Client Version: 514
-        Server Revision: 34bf3ff6a39a09848d5552e0486c0f0fedb58fc9 - dev - 2022-08-06
-        Game ID: cji-cImG
-        Current map: SEV Torch
+      label: Шаги для повторения проблемы.
+      description: Самая важная часть. Опишите ВСЁ, что вы делали, что бы встретиться с проблемой. (Обязательное поле)
+      placeholder: Find a chair, buckle yourself into it, be eaten by dragon.
```

В этот раз хоть бы не допустить ошибку
```suggestion
      placeholder: Find a chair, buckle yourself into it, be eaten by dragon.
```